src/lib/rest/rest.cpp

```diff
@@ -922,14 +924,19 @@ static ApiVersion apiVersionGet(const char* path)
     return V2;
   }
 
-  // Different from v2, v1 is case-insensitive (see case/2057 test)
```

Comment:
I understand that the changes in apiVersionGet() at the end are just a refactor to put /log in a difrerent if block but that the way the function works doesn't change. Is this correct?